.github/workflows/build-and-test.yml

```diff
@@ -419,7 +419,7 @@ jobs:
       fail-fast: false
```

Comment:
I missed this the first time, but is the reason for having a separate build for explorer in this file because you need the build binaries and explorer can't be installed like ttrt does? (aka a whls)

Reply:
It probably could be installed as a whl, but explorer depends on most of tt-mlir and will always be used with it. We don't have a usecase for it as as standalone build which is why we don't need the whl.

This does make the job repeat tt-mlir build steps, but i don't see a clean way of reusing these for explorer. Any thoughts?

Reply:
so I think it'll probably be faster to just build it from source and run it in the same test flow, but a longer tt-explorer whls solution would make sense since you can effectively decouple it from compiler and ttrt already pip cleans anything you need from runtime

Reply:
TT-explorer uses the ttir_to_ttnn_backend_pipeline / ttnn_to_flatbuffer_file flows and ttrt, so it essentially depends on the whole compiler. That's why i don't see a usecase in decoupling it and building as a standalone.